flair/models/text_classification_model.py

```diff
@@ -258,6 +262,14 @@ def _get_single_label(self, label_scores) -> List[Label]:
 
         return [Label(label, conf.item())]
 
+    def _predict_label_probab(self, label_scores) -> List[Label]:
```

Comment:
I would recommend to name the function `_predict_label_prob` (`prob` is more often used than `probab`)

Reply:
Got it. Sure